src/lib/components/chipList/chip/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+
+import styles from './style.module.scss';
+import removeButtonIcon from '../icons/remove-button.svg';
+import removeButtonHighlightedIcon from '../icons/remove-button-highlighted.svg';
+import { Option } from '../../../models/suggestMultiSelect';
+
+export default ({
```

Comment:
```suggestion
const Chip = ({
```

Maybe the chip is a component of its own that can be used as-is?